pom.xml

```diff
@@ -13,6 +13,12 @@
   <description>
     This is the top level project that builds, packages the core and all the DB bindings for YCSB infrastructure.
   </description>
+
+  <scm>
+    <connection>scm:git:git://github.com/someuser/somelibrary.git</connection>
```

Comment:
Excellent catch. Should have been more careful when replacing my original git@github ssh connection.